packages/wrangler/src/deploy/deploy.ts

```diff
@@ -76,6 +76,7 @@ type Props = {
 	compatibilityDate: string | undefined;
 	compatibilityFlags: string[] | undefined;
 	assetPaths: AssetPaths | undefined;
+	experimentalAssets: string | undefined;
```

Comment:
This prop has been added but there are no changes to the implementation of `deploy()`? Is that correct?

Reply:
Ah I see this PR is just adding the flag and validation

Reply:
yes that is correct. This just preps the boilerplate for us, to when we're ready to start implementing the new behaviour